docs/database/database-spans.md

```diff
@@ -1,5 +1,5 @@
 <!--- Hugo front matter used to generate the website version of this page:
-linkTitle: Client Calls
+linkTitle: Spans
```

Comment:
Will "Spans" here be a link title inside a folder named "Database"?

Reply:
Yes